src/usb_reset/include/usb_reset/owr_ros_components.hpp

```diff
@@ -0,0 +1,22 @@
+/*
+ * Date Started: 25/07/16
+ * Original Author: Harry J.E Day
+ * Editors:
+ * Purpose: C++ code representation of the QML package
+ * This code is released under the MIT License. Copyright BLUEsat UNSW, 2017
+ */
+#ifndef OWR_ROS_COMPONENTS_H
+#define OWR_ROS_COMPONENTS_H
+
+#include <QQmlExtensionPlugin>
+
+
+class OWR_ROS_Components : public QQmlExtensionPlugin {
```

Comment:
If you are creating a new version of this it needs to have a different name then the old one (Do you NEED to create a new one?)